README.md

```diff
@@ -945,6 +945,49 @@ container_pull(
 
 This can then be referenced in `BUILD` files as `@base//image`.
 
+If you wish to use container_pull using custom docker authentication credentials,
```

Comment:
add a note in line 1168:
Note: container_pull now supports using custom docker auth, see sample
(same for container_push declaration somewhere below)

Reply:
Done